<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Metronome</class>
 <widget class="QDialog" name="Metronome">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>227</width>
    <height>161</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Calibri</family>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="metronome_toggle">
     <property name="toolTip">
      <string>Toggle the metronome on and off</string>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="bpm_input">
     <property name="font">
      <font>
       <pointsize>32</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>View and enter the BPM of the metronome</string>
     </property>
     <property name="maxLength">
      <number>3</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>BPM</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="bpm_adjustment_container">
     <item>
      <widget class="QPushButton" name="decrease_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Decrease the BPM of the metronome</string>
       </property>
       <property name="accessibleName">
        <string>Decrease BPM</string>
       </property>
       <property name="text">
        <string>▼</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="bpm_blinker" native="true">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: black;
border-radius: 10px;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="increase_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Increase the BPM of the metronome</string>
       </property>
       <property name="accessibleName">
        <string>Increase BPM</string>
       </property>
       <property name="text">
        <string>▲</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
